ports/zoe/portfile.cmake

```
@@ -0,0 +1,39 @@
+vcpkg_from_github(
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO winsoft666/zoe
+    HEAD_REF master
+    REF 8241475c5e5bd044738bf58adbe55afa366e81f2
+    SHA512 c7552dbc154f62cf83a52adfa30ff5bd8e1af0c99aa40a122175d87b08d578c847076416712d293102e8554097807574c2d5d9f7bdc2dfce024556e9d65c3890
+)
+
+string(COMPARE EQUAL "${VCPKG_LIBRARY_LINKAGE}" "static" LIBRARY_LINKAGE_HAS_STATIC)
+string(COMPARE EQUAL "${VCPKG_CRT_LINKAGE}" "static" ZOE_USE_STATIC_CRT)
+
+if(${LIBRARY_LINKAGE_HAS_STATIC})
+	set(ZOE_BUILD_SHARED_LIBS OFF)
+else()
+	set(ZOE_BUILD_SHARED_LIBS ON)
+endif()

```

```suggestion
string(COMPARE EQUAL "${VCPKG_LIBRARY_LINKAGE}" "dynamic" ZOE_BUILD_SHARED_LIBS)
string(COMPARE EQUAL "${VCPKG_CRT_LINKAGE}" "static" ZOE_USE_STATIC_CRT)
```

BTW vcpkg already set `BUILD_SHARED_LIBS`, the standard variable.